### ISSUE TITLE
Move WebGME example to here

### ISSUE THREAD

**Issue by pgleeson** [ModECI/MDF#66]
From https://github.com/ModECI/MDFTests/tree/main/mdf_gme to WebGME dir when #64 is merged

**Comment by pgleeson** [comment]
@mraunak Do you want to look into doing this? It might just be a PR with a copy of all of the files from the dir mentioned above, but it would be good to go through the README there yourself to see if it's all clear.

**Comment by mraunak** [comment]
@pgleeson Yeah, Sure I will go through the README and check it. Will let you know if face any issue.

**Comment by mraunak** [comment]
@ pgleeson by mistake it got closed, I am reopening it.
